### ISSUE TITLE
train sam3

### ISSUE THREAD

**Issue by ruiyangjin666** [facebookresearch/sam3#277]
"For some images without masks, the loss becomes 0 during training. How should this be configured? In other words, is this a problem of handling negative samples?

**Comment by t2tu** [comment]
I want to fine-tune SAM3 using images from other datasets and category texts as inputs. I wonder if you have successfully completed the training, and whether I can refer to your configuration file?
